sda/internal/config/config.go

```diff
@@ -192,29 +192,29 @@ func NewConfig(app string) (*Config, error) {
 	if viper.IsSet("configFile") {
 		viper.SetConfigFile(viper.GetString("configFile"))
 	}
-	log.Infoln("reading config")
+	log.Debugf("Attempting to read configuration...")
 	if err := viper.ReadInConfig(); err != nil {
-		log.Infoln(err.Error())
+		log.Debugf("Error reading config : %v", err.Error())
 		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			log.Infoln("ReadInConfig Error")
+			log.Warnf("ReadInConfig Error : %v", err.Error())
 
 			return nil, err
 		}
-		log.Infoln("No config file found, using ENVs only")
+		log.Debugf("No config file found, using ENVs only")
```

Comment:
I imagine that this could be useful to have as info..?

Reply:
I think this is debug info. The problem is that should we debug the production server when there is a problem? In principle, debugging should be done with the staging server which has log level set to debug, but in reality...

Reply:
I think this is debug info. The problem is that should we debug the production server when there is a problem? In principle, debugging should be done with the staging server which has log level set to debug, but in reality...